pages/data-providers/[data-provider-id]/deposit-compliance.module.css

```
@@ -15,12 +15,24 @@
   }
 }
 
-.compliance-level {
+.data-overview {
   flex: 2 2 60%;
   width: 100%;
 }
 
+.numbers {
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 1rem;
+
+  & > p {
+    flex: 50%;
+    margin: 0;
```

The following may not pass linter. I am trying to explicitly set flex parameters (grow is 0 by default) and add inter-column spacing just in case.

```suggestion
    flex: 1 0 50%;
    margin: 0 var(--component-padding-x, 1rem) var(--component-padding-y, 1rem);
```